templates/test-sw.hbs

```diff
@@ -44,22 +44,22 @@
 
     <script>
       mocha.setup({
-        ui: 'bdd'
+        ui: 'bdd',
+        timeout: 10000,
       });
 
       chai.should();
-      const expect = chai.expect;
+      window.expect = chai.expect;
     </script>
 
     <!-- Package-specific tests matching packages/<pkg>/test/browser/*.js -->
 
     <script>
     describe('{{pkg}} > test/sw/*.js', function() {
-      {{#each scripts}}
-      it(`should run '{{this}}' sw tests`, function() {
```

Comment:
You have the use case for the bundling, my gut would be coming up with something than makes sense to you and following it.

Off the top of my head, have "bundled" under browser and sw directories, ignoring those from the test list, building them into browser and sw respectively as _bundled.js or something similar and making sure they are ignored with gitignore.